ecs-image-build/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG IMAGE_VERSION="latest"
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-open-jdk8-maven-build:${IMAGE_VERSION}
+
+ARG S3_RELEASE_BUCKET=""
+ARG APPLICATION_NAME=docs.developer.ch.gov.uk
+ARG APPLICATION_VERSION=""
+ARG AWS_ACCESS_KEY_ID=""
+ARG AWS_SECRET_ACCESS_KEY=""
+ARG AWS_REGION="eu-west-2"
+ARG FILE_NAME=""
+ARG PORT="8080"
+ARG JAVA_MEM_ARGS=""
+
+WORKDIR /opt
+
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
```

Comment:
I think we need to find a better way of handling this. The aws cli should not be required to be installed in every production docker image and we should not have to download the zip file directly from s3 as part of this image being built.

We should be able to build this image just by copying binaries/assets from the local file system and not relying on the zip being in s3.

If we are 100% set on getting the zip from s3 for the binaries/assets (rather than copying it around within concourse) then it should be done (and unzipped) before this docker image is built

Reply:
Changes have been made in the pipeline and dockerfile to unarchive the artifact in concourse before the docker build